Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -993,13 +1006,24 @@ void InspectorPageAgent::defaultUserPreferencesDidChange()
 
     defaultUserPreferences->addItem(prefersContrastUserPreference);
 
+#if ENABLE(DARK_MODE_CSS) || HAVE(OS_DARK_MODE_SUPPORT)
+    bool useDarkAppearance = m_inspectedPage.defaultUseDarkAppearance();
+    auto prefersColorSchemeUserPreference = Protocol::Page::UserPreference::create()
+        .setName(Protocol::Page::UserPreferenceName::PrefersColorScheme)
+        .setValue(useDarkAppearance ? Protocol::Page::UserPreferenceValue::Dark : Protocol::Page::UserPreferenceValue::Light)
+        .release();
+
+    defaultUserPreferences->addItem(prefersColorSchemeUserPreference);
+#endif
+
     m_frontendDispatcher->defaultUserPreferencesDidChange(WTFMove(defaultUserPreferences));
 }
 
 #if ENABLE(DARK_MODE_CSS) || HAVE(OS_DARK_MODE_SUPPORT)
 void InspectorPageAgent::defaultAppearanceDidChange(bool useDarkAppearance)
 {
-    m_frontendDispatcher->defaultAppearanceDidChange(useDarkAppearance ? Protocol::Page::Appearance::Dark : Protocol::Page::Appearance::Light);
+    defaultUserPreferencesDidChange();
+    UNUSED_PARAM(useDarkAppearance);
 }
```

Comment:
Instead of marking the parameter as unused, we should remove the parameter entirely from here, the InspectorInstrumentation, and anyone calling InspectorInstrumentation.

Reply:
Removed all instances of the unused parameter along the call chain. I meant to do this and just put this in as a shim, but it seems I forgot about it. Cleaned up now.